package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/answers-headless",
-  "version": "1.1.0-beta.12",
+  "version": "1.1.1-alpha.0",
```

Comment:
I think we decided that alpha/beta tags should include the commit date in the name?

Reply:
Will update after the pr is ready to publish (although now I'm thinking if we want a commit hash instead of a date instead)

Reply:
Would the PR number be better since it's known before merge and gives more granularity than commit date?

Reply:
I'm thinking potentially a github action that checks a pr commit message after merging to develop for a specific string (maybe `PUBLISH='1.1.1-alpha.0'`) where it check in develop branch, update the package version locally, and publish an alpha version, maybe make an alpha branch too.